services/harmony/package.json

```diff
@@ -99,7 +99,7 @@
     "express-winston": "^4.1.0",
     "fastq": "^1.11.0",
     "form-data": "^4.0.0",
-    "gdal-async": "~3.7.2-1",
+    "gdal-async": "^3.8.3",
```

Comment:
We'll want to test this one in a sandbox environment. It might require us changing to install a different version of gdal (and potentially building from source and saving it in the s3 bucket in each account). I think you should also lock the minor version (similar to what was there before with the ~) since gdal minor updates usually require a different version of gdal installed.

Reply:
This update is not necessary anyways as it is not resolve the vulnerability. I will remove this update.